tests/main/views/test_direct_award.py

```diff
@@ -0,0 +1,497 @@
+import json
+from datetime import datetime
+import math
+import random
+
+import pytest
+from tests.helpers import FixtureMixin
+from tests.bases import BaseApplicationTest
+
+from dmapiclient.audit import AuditTypes
+from app.models import DATETIME_FORMAT, AuditEvent, User
+from app.models.direct_award import ProjectUser, Search, Project
+from ...helpers import DIRECT_AWARD_SEARCH_URL, DIRECT_AWARD_PROJECT_NAME
+
+
+class DirectAwardSetupAndTeardown(BaseApplicationTest, FixtureMixin):
+    def setup(self):
+        super(DirectAwardSetupAndTeardown, self).setup()
+        self.user_id = self.setup_dummy_user(id=1, role='buyer')
+        self.direct_award_project_name = DIRECT_AWARD_PROJECT_NAME
+        self.direct_award_search_url = DIRECT_AWARD_SEARCH_URL
+
+    def teardown(self):
+        super(DirectAwardSetupAndTeardown, self).teardown()
```

Comment:
Does this actually do anything? If we don't override the teardown function, it's still available to get called, right? Or is there something about the multiple inheritance thing that I'm forgetting...